ports/atmel-samd/boards/picoplanet/mpconfigboard.h

```diff
@@ -0,0 +1,69 @@
+#define MICROPY_HW_BOARD_NAME "PicoPlanet"
+#define MICROPY_HW_MCU_NAME "samd21e18"
+
+
+
+#define MICROPY_PORT_A        (PORT_PA00 | PORT_PA01)
+#define MICROPY_PORT_B        (0)
+#define MICROPY_PORT_C        (0)
+
+#define IGNORE_PIN_PA00     1
+#define IGNORE_PIN_PA01     1
+#define IGNORE_PIN_PA10     1
+#define IGNORE_PIN_PA11     1
+#define IGNORE_PIN_PA12     1
+#define IGNORE_PIN_PA13     1
+#define IGNORE_PIN_PA14     1
+#define IGNORE_PIN_PA15     1
+#define IGNORE_PIN_PA18     1
+#define IGNORE_PIN_PA19     1
+#define IGNORE_PIN_PA20     1
+#define IGNORE_PIN_PA21     1
+#define IGNORE_PIN_PA22     1
+#define IGNORE_PIN_PA23     1
+// USB is always used internally so skip the pin objects for it.
+#define IGNORE_PIN_PA24     1
+#define IGNORE_PIN_PA25     1
+#define IGNORE_PIN_PA27     1
+#define IGNORE_PIN_PA28     1
+//#define IGNORE_PIN_PA30     1
+#define IGNORE_PIN_PA31     1
+#define IGNORE_PIN_PB01     1
+#define IGNORE_PIN_PB02     1
+#define IGNORE_PIN_PB03     1
+#define IGNORE_PIN_PB04     1
+#define IGNORE_PIN_PB05     1
+#define IGNORE_PIN_PB06     1
+#define IGNORE_PIN_PB07     1
+#define IGNORE_PIN_PB08     1
+#define IGNORE_PIN_PB09     1
+#define IGNORE_PIN_PB10     1
+#define IGNORE_PIN_PB11     1
+#define IGNORE_PIN_PB12     1
+#define IGNORE_PIN_PB13     1
+#define IGNORE_PIN_PB14     1
+#define IGNORE_PIN_PB15     1
+#define IGNORE_PIN_PB16     1
+#define IGNORE_PIN_PB17     1
+#define IGNORE_PIN_PB22     1
+#define IGNORE_PIN_PB23     1
+#define IGNORE_PIN_PB30     1
+#define IGNORE_PIN_PB31     1
+#define IGNORE_PIN_PB00     1
```

Comment:
You don't need these because these pins are not even defined for SAMD21E variant.